README.md

```diff
@@ -170,11 +177,40 @@ The `ingestion-data/dataset-config/` directory holds json files that can be used
         }
     ]
 }
+
+```
+
+</details>
+
+### `production/transfer-config`
+
+This directory contains the configuration needed to execute a stand-alone airflow DAG that copies data from a specified staging bucket and prefix to a permanent location in `s3://veda-data-store` using the collection_id as a prefix.
+
+Should follow the following format:
+
+<details>
+  <summary><b>/production/transfer-config/collection_id.json</b></summary>
+
+```json
+{
+    "collection": "<collection-id>",
+
+    ## the location of the staged files
+    "origin_bucket": "<s3-bucket>",
+    "origin_prefix": "<s3-key-prefix>",
+    "bucket": "<s3-bucket>",
+    "filename_regex": "<filename-regex>",
+
+    ### misc
+    "dry_run": "<true/false>"
+}
 ```
 
+</details>
+
 ## Validation
```

Comment:
@anayeaye as part of the validation should we also create json schema that includes our minimum requirements for collection metadata to maintain consistency across the production catalog? For example, all collections in VEDA must include:

- providers
- thumbnails assets

Reply:
Ooof I just included those there in an optional section in the updated geoglam notebook that is now runnable in the hub https://github.com/NASA-IMPACT/veda-docs/blob/c9dc33a8f2960fa29a59db2974f979b12055f8ca/contributing/dataset-ingestion/example-template/example-geoglam-ingest.ipynb

Reply:
I don't disagree on a firm nudge to create renders and thumbnails but I don't think we should require them until we have great docs on how to set them up + a way for users to upload thumbs (which are both doable)